src/EnergyPlus/DataRoomAirModel.hh

```diff
@@ -153,185 +163,111 @@ namespace DataRoomAirModel {
         Num
     };
 
+    extern const std::array<std::string_view, (int)Comfort::Num> comfortNamesUC;
+        
     struct AirModelData
     {
         // Members
-        std::string AirModelName;
-        std::string ZoneName;
-        int ZonePtr;               // Pointer to the zone number for this statement
-        RoomAirModel AirModelType; // 1 = Mixing, 2 = Mundt, 3 = Rees and Haves,
-        // 4 = UCSDDV, 5 = UCSDCV, -1 = user defined
-        // 6 = UCSDUFI, 7 = UCSDUFE, 8 = AirflowNetwork
-        CouplingScheme TempCoupleScheme; // 1 = absolute (direct),
-        // 2 = relative air model temperature passing scheme (indirect)
-        bool SimAirModel; // FALSE if Mixing air model is currently used and
+        std::string Name = "";
+        std::string ZoneName = "";
+        int ZonePtr = 0;               // Pointer to the zone number for this statement
+        RoomAirModel AirModel = RoomAirModel::Mixing; 
+        CouplingScheme TempCoupleScheme = CouplingScheme::Direct; 
+        bool SimAirModel = false; // FALSE if Mixing air model is currently used and
         // TRUE if other air models are currently used
-
-        // Default Constructor
-        AirModelData() : ZonePtr(0), AirModelType(RoomAirModel::Mixing), TempCoupleScheme(CouplingScheme::Direct), SimAirModel(false)
-        {
-        }
     };
 
     struct AirNodeData
     {
         // Members
-        std::string Name; // name
-        std::string ZoneName;
-        int ZonePtr;               // Pointer to the zone number for this statement
-        AirNodeType ClassType;     // depending on type of model
-        Real64 Height;             // height
-        Real64 ZoneVolumeFraction; // portion of zone air volume associated with this node
+        std::string Name = ""; // name
+        std::string ZoneName = "";
+        int ZonePtr = 0;               // Pointer to the zone number for this statement
```

Comment:
Maybe for a different pass since this PR is rather intense, I see ZoneName and ZonePtr in these and other structs. If we know the zone index we also know the name.

Reply:
Yeah.